plugins/WatchStateLoggerPlugin.ts

```diff
@@ -42,7 +43,28 @@ export class WatchStateLoggerPlugin {
 
             process.send && process.send(messages.compilationComplete, error => null);
             // Send emitted files so they can be LiveSynced if need be
-            process.send && process.send({ emittedFiles: emittedFilesFakePaths }, error => null);
+            process.send && process.send({ emittedFiles: emittedFilesFakePaths, webpackRuntimeFiles: runtimeOnlyFiles }, error => null);
         });
     }
 }
+
+function getWebpackRuntimeOnlyFiles(compilation, compiler) {
+    let runtimeOnlyFiles = [];
+    try {
+        runtimeOnlyFiles = [].concat(...compilation.chunkGroups
```

Comment:
Maybe you should get the `compilation.entrypoints` instead of `chunkGroups`? https://github.com/webpack/webpack/blob/8a7597aa6eb2eef66a8f9db3a0c49bcb96022a94/lib/optimize/RuntimeChunkPlugin.js#L20